src/front/glsl/builtins.rs

```diff
@@ -537,7 +537,7 @@ fn inject_standard_builtins(
         | "dFdyFine" | "dFdyCoarse" | "fwidth" | "fwidthFine" | "fwidthCoarse" => {
             // bits layout
             // bit 0 trough 1 - dims
-            for bits in 0..(0b100) {
+            for bits in 0..0b100 {
```

Comment:
What's up with all these changes?

Reply:
Clippy also started flagging these as unused parentheses so I just applied the suggestion